products/views.py

```diff
@@ -1,3 +1,42 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.core.exceptions import MultipleObjectsReturned
+from django.http            import JsonResponse
+from django.views           import View
+
+from .models      import Food, Category, FoodImage, RequiredOption, SelectOption
+
+class FoodView(View):
+    def get(self, request, food_id):
+        try:
+            food             = Food.objects.get(id=food_id)
```

Comment:
너무 많은 인덴트는 가독성을 안좋게 만듭니다.

```suggestion
            food = Food.objects.get(id=food_id)
```